packages/clarity/clarity.0.3.1/opam

```diff
@@ -0,0 +1,14 @@
+opam-version: "1.2"
+version: "0.3.1"
+maintainer: "joyprophecy@gmail.com"
+author: "joyprophecy@gmail.com"
+homepage: "https://github.com/IndiscriminateCoding/clarity"
+bug-reports: "https://github.com/IndiscriminateCoding/clarity/issues"
+dev-repo: "https://github.com/IndiscriminateCoding/clarity.git"
+license: "BSD3"
+available: [ ocaml-version >= "4.04.0" ]
+depends: [ "jbuilder" {build} "ocamlfind" {build} ]
```

Comment:
I don't think you need ocamlfind here. Jbuilder should be enough.

Reply:
Is it a blocker?

Reply:
No, but please think of it next time.